build.sbt

```diff
@@ -1,27 +1,35 @@
 import PlayCrossCompilation.{dependencies, version}
 import play.core.PlayVersion
 
+val scala2_12               = "2.12.8"
+val scala2_13               = "2.13.7"
 lazy val majorVersionNumber = 0
 
+def scalacOptionsVersion(scalaVersion: String) =
+  CrossVersion.partialVersion(scalaVersion) match {
+    case Some((2, 12)) => Seq("-Ypartial-unification")
```

Comment:
cool, googled and this is on by default now, leaving comment in case I forget later or anyone is unsure if we checked

Reply:
I should have commented on the ticket myself, thanks